app/viewmodels/DutyDefermentAccountViewModel.scala

```diff
@@ -146,20 +148,18 @@ object DutyDefermentAccountViewModel {
           None
         }
 
-        val headWithEntriesOrNoStatement = if (statement.groups.head.monthAndYear.compareTo(monthsToDisplay) > 0) {
+        val headWithEntriesOrNoStatement = if (statement.groups.size > 0) {
           val ddHead = new duty_deferment_head(emptyH2Component, emptyPComponent).apply(statement.groups.head)
 
           val entries: Seq[HtmlFormat.Appendable] = statement.groups.tail.map {
             entry =>
-              val statementPeriods = if (entry.monthAndYear.compareTo(monthsToDisplay) > 0) Seq(entry) else Seq()
+              val statementPeriods = Seq(entry)
```

Comment:
If we no longer need the condition we can likely remove this entire line and swap statementPeriods on Line 157 with Seq(entry) as below:

new duty_deferment_accordian(new GovukAccordion()).apply(Seq(entry), 0)